TSC-Charter.md

```diff
@@ -125,9 +125,11 @@ The TSC voting members follow a [Consensus Seeking][] decision making model.
 When an agenda item has appeared to reach a consensus the moderator will ask
 "Does anyone object?" as a final call for dissent from the consensus.
 
-For all votes, a simple majority of all TSC voting members for, or against, the
-issue wins. A TSC voting member may choose to participate in any vote through
-abstention.
+For all votes, the winning candidate option is the one that wins a simple
+majority of all TSC voting members in every head-to-head election against each
```

Comment:
```suggestion
majority of all participated / voted TSC voting members in every head-to-head election against each
```

Reply:
a small suggestion on the scope majority - among those who voted. (not a blocker)